lrlex/src/lib/builder.rs

```diff
@@ -84,6 +87,13 @@ where
         self
     }
 
+    /// Set if the generated module should be public. This is only necessary if
```

Comment:
I'm not sure I understand the "this is only necessary" sentence: surely it's useful only *if* you use `lrlex_mod!`?